src/middleware/CompanyAuthMiddleware.ts

```diff
@@ -41,7 +42,8 @@ export class CompanyAuthMiddleware extends BaseMiddleware {
 
         const appeal: Appeal = applicationData.appeal;
         if (!appeal) {
-            return next(new Error('Appeal data not found'));
+            loggerInstance().info(`CompanyAuthMiddleware: Appeal data not found in session, redirecting to ${PENALTY_DETAILS_PAGE_URI}`);
+            return res.redirect(PENALTY_DETAILS_PAGE_URI);
```

Comment:
If no appeal data is found at this stage, shouldnt be throw an error and fail fast?

Reply:
we previously failed fast and displayed a 500 error page, but then we thought it would be a better user experience if we instead redirected them to the beginning of the journey when no appeal data